config/services.php

```diff
@@ -35,4 +35,10 @@
         'secret' => env('STRIPE_SECRET'),
     ],
 
+    'google' => [
+        'maps' => [
+            'key' => env('GOOGLE_MAPS_API_KEY')
```

Comment:
Also, this is complex enough to get (if I remember correctly) that we should add instructions to the README for how to get them